### PR TITLE
Handling HSTS database file

### PR DESCRIPTION
**Pull request by Martinski4GitHub**
Added code to prevent the error message about the HSTS database file.

### CODE REVIEW

**Comment by Martinski4GitHub** [comment]
@ExtremeFiretop,

This PR also includes code to be able to "reset the lock file" on demand.
This is really meant only for us (for testing & debugging purposes).
I had this code in my "debug" version for a while and I thought I'd put it in the "official" version as well.

**Comment by ExtremeFiretop** [comment]
> @ExtremeFiretop,
> 
> This PR also includes code to be able to "reset the lock file" on demand. This is really meant only for us (for testing & debugging purposes). I had this code in my "debug" version for a while and I thought I'd put it in the "official" version as well.

Ha, the amount of times I've manually deleted that lock file. Thank you for making it one step easier on us! Haha

File: MerlinAU.sh [hunk 1]
```diff
@@ -2899,6 +2899,14 @@ Please manually update to version $minimum_supported_version or higher to use th
         Say "Latest release version is ${GRNct}${release_version}${NOct}."
         Say "Downloading ${GRNct}${release_link}${NOct}"
         echo
+
+        ##----------------------------------------##
+        ## Modified by Martinski W. [2024-Feb-28] ##
+        ##----------------------------------------##
+        # Avoid error message about HSTS database #
+        wgetHstsFile="/tmp/home/root/.wget-hsts"
```
Comment (ExtremeFiretop) [inline MerlinAU.sh:2907]: Approved.

File: MerlinAU.sh [hunk 2]
```diff
@@ -3594,11 +3602,19 @@ keepZIPfile=0
...
 
 ##----------------------------------------##
-## Modified by Martinski W. [2023-Dec-26] ##
+## Modified by Martinski W. [2024-Feb-28] ##
 ##----------------------------------------##
 # Prevent running this script multiple times simultaneously #
 if ! _AcquireLock_
-then Say "Exiting..." ; exit 1 ; fi
+then
+    if [ $# -eq 1 ] && [ "$1" = "resetLockFile" ]
```
Comment (ExtremeFiretop) [inline MerlinAU.sh:3610]: Approved

**Comment by ExtremeFiretop** [comment]
Tested with success. Merged. Successfully suppressed the HSTS warning.

**Pull request merged**